docs/snphost.1.adoc

```diff
@@ -99,12 +99,33 @@ COMMANDS
  options:
     -h, --help          Show a help message
 
-*snphost vcek*::
-	usage: snphost vcek [ der, pem ] FILE-PATH
+*snphost fetch ca*::
+	usage: snphost fetch ca [ der, pem ] FILE-PATH
 
-        This command fetches the host system's VCEK and writes the encoded
-        certificate to the file at path FILE-PATH. User must specify which
-        format they would like the certificate to be encoded in (DER or PEM).
+        This command fetches the host system's certificate chain  and writes
+		the encoded certificates to the file at path FILE-PATH. Users must
+		specify which format they would like the certificate to be encoded
+		in (DER or PEM).
+
+ options:
+    -h, --help          Show a help message
+
+*snphost fetch vcek*::
+	usage: snphost fetch vcek [ der, pem ] FILE-PATH
+
+        This command fetches the host system's versioned chip endorsement
+		key (VCEK) and writes the encoded certificate to the file at path
+		FILE-PATH. Users must specify which format they would like the
+		certificate to be encoded in (DER or PEM).
+
+ options:
+    -h, --help          Show a help message
+
+*snphost fetch crl*::
+	usage: snphost fetch crl FILE-PATH
```

Comment:
Same here, according to the code this should be a DIR-PATH. But in this case maybe we want to make it a file? You're only pulling one .crl file. Also, I'm still trying to completely understand how the CRLs work, but would there be any interest in allowing users to pull the crl of any processor generation they want. Maybe as an option, and default it to their current processor generation. That way they don't need a Genoa to pull a Genoa CRL. Idk if there is a use-case for that, but just a thought.

Reply:
I think in this case it makes sense to only stick with platform in question. @tylerfanelli do you have an opinion about how this should behave?

Reply:
The platform currently being used seems fine to me. However. in other cases we do give the option for the generation.